charts/keycloak/templates/keycloak_db.yaml

```
@@ -1,5 +1,18 @@
 {{- if .Values.keycloak.database.create }}
 ---
+{{- if .Values.keycloak.pvc.enabled }}
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: postgresql-pvc
+spec:
+  resources:
+    requests:
+      storage: {{ .Values.keycloak.pvc.size }}
+  accessModes:
+  - ReadWriteOnce
+---
+{{- end }}
```

```suggestion
{{- if .Values.keycloak.pvc.enabled }}
---
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  name: postgresql-pvc
spec:
  resources:
    requests:
      storage: {{ .Values.keycloak.pvc.size }}
  accessModes:
  - ReadWriteOnce
{{- end }}
---
```

I think the `---` make more sense this way :)